app/views/home/index.html.haml

```diff
@@ -33,7 +33,7 @@
     %ol.list--block
       %li
         .list--details
-          %h4= link_to t('search_s').capitalize, '/search'
+          %h4= link_to(t('search_s').capitalize, '/search')
```

Comment:
Could you replace the fixed path '/search' with the routes helper **search_path**